File: cpp/bc1116.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
	int fVal, sVal, cont;
	
	cin >> cont;
	
	for(int i = 0; i < cont; i++)
	{
		cin >> fVal >> sVal;
		
		if(sVal == 0) cout << "divisao impossivel" << endl;
		else cout << fixed << setprecision(1) << (float) fVal/ sVal << endl;
	}
}

File: python/bc1116.py
cont = int(input())

for c in range(0, cont):
    cVal = input().split(" ")

    fVal = int(cVal[0])
    sVal = int(cVal[1])

    if sVal == 0: print("divisao impossivel")
    else: print(f"{fVal / sVal:.1f}")
